test/system/070-build.bats

```diff
@@ -210,6 +210,40 @@ EOF
     run_podman rmi -f build_test
 }
 
+@test "podman parallel build should not race" {
+    rand_filename=$(random_string 20)
+    rand_content=$(random_string 50)
+    #iterator
+    iter=1
+
+    tmpdir=$PODMAN_TMPDIR/build-test
+    mkdir -p $tmpdir
+    mkdir -p $PODMAN_TMPDIR/reldir
+
+    containerfile=$PODMAN_TMPDIR/reldir/Containerfile
+    cat >$containerfile <<EOF
+FROM quay.io/libpod/alpine:latest
+RUN echo $rand_content > /$rand_filename
+EOF
+
+    run_podman rm --all --force --ignore
+    run_podman rmi -af
+    while [ $iter -le 30 ]
+    do
+      run_podman build -t $iter -f $containerfile &> /dev/null
```

Comment:
This &>/dev/null is a nop

Reply:
@edsantiago Reverting to older racy `SetNames` fails this test. But if you can suggest a better way to move this to background in `bats` i'll amend this.  Thanks for taking a look at this test.

Reply:
I do not understand this comment. I have run your test (and mine) against main @ ed59b89a4 and I cannot get the test to fail. (I can't get mine to fail, either, which is why I think I don't understand the problem)

Reply:
@edsantiago No issues, anyways it should pass with the new test which you have added as well. I have verified it passes with new test which you just shared.